### ISSUE TITLE
Websocket is losing connection / data

### ISSUE THREAD

**Issue by tlvenn** [quarkusio/quarkus#5732]
**Describe the bug**
Simply following the websocket guide with 2-3 tabs in the browser.
Very quickly after pushing some messages, the following happens on Quarkus:
![quarkus_websocket_bug](https://user-images.githubusercontent.com/3340262/69503982-b40e2f80-0f6a-11ea-8623-46d850312c22.png)

And from there, weird behaviors happen in the chat (messages are lost, etc...)
Reverting to Quarkus 0.22.0 and the issue completely disappear.
So it seems there is an issue between the integration of Vertx & Undertow with websocket.

**Expected behavior**
No ClosedChannelException is emitted, websockets don't close by themselves

**Actual behavior**
Websockets are been closed or channels are lost / not routed properly. I am not exactly sure what is happening, I did not investigate it further.

**To Reproduce**
1. Simply follow the websocket guide with Quarkus 1.0.0
2. Open 2 tabs in your browser
3. Send a few messages

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: OSX 10.15
 - Output of `java -version`: Java version "1.8.0_181"
 - Quarkus version: 1.0.0

**Comment by stuartwdouglas** [comment]
I just tested this out and could not reproduce it, in either dev or native mode.

**Comment by jaikiran** [comment]
@tlvenn Just curious - which vendor and version of the browser are you using?

**Comment by tlvenn** [comment]
Firefox 70.0.1 but that should not be the culprit, we have a java desktop app which uses websocket and when we migrated the server endpoint to quarkus, the connections were not stable with the same errors.

This is when I tried the example to see if I could reproduce it and I was able to without any issue but I did that in our existing project not from a scratch one.

So I will try again from scratch and verify I can still reproduce this. It might be that there are a few pieces from our project which create the issue. For example we have a web.xml and we are using Undertow predicate language.
